notebooks/290-stable-diffusion-keras-cv/290-stable-diffusion-keras-cv.ipynb

```diff
@@ -0,0 +1,722 @@
+{
```

Comment:
<p>I belive it should be before any other descriptions, not inside conversion block. Usually we separate dependencies installation into prerequisites block (sometimes, users want to run notebooks several times like demo, but having it inside conversion block, makes feeling that all of them need only for conversion. Also I think note about python requirement also should be more highlighted, if it is not a part of conversion section</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1836/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Prerequisites block added. Any ideas on additional python 3.9 note highlighting?</p>

Reply:
not sure that it works for all possible preview format, maybe we can make it using [!NOTE] markdown syntax:
https://github.com/sinsukehlab/NOTE-test/blob/main/NOTE.md
?

Reply:
another possible way is using html 
https://gist.github.com/DanielKotik/4b81480c479a57e0dd13ac4d153e4451